crates/chia-tools/src/bin/test-block-generators.rs

```diff
@@ -158,17 +177,25 @@ fn main() {
 
                 // after the hard fork, we run blocks without paying for the
                 // CLVM generator ROM
-                let block_runner = if height >= 5_496_000 {
+                let block_runner = if args.original_generator || height >= args.hard_fork_height {
```

Comment:
```suggestion
                let block_runner = if !args.original_generator || height >= args.hard_fork_height {
```

Reply:
I don't think this is correct. the point of `--original-generator` is to compare the original one to the hard-fork one. The original one is called on line 211, so we need to use the hard-fork one here